containers/jetty-http/pom.xml

```diff
@@ -73,29 +73,11 @@
         </plugins>
 
         <resources>
-            <resource>
-                <directory>${basedir}/src/main/java</directory>
-                <includes>
-                    <include>META-INF/**/*</include>
-                </includes>
-            </resource>
             <resource>
                 <directory>${basedir}/src/main/resources</directory>
                 <filtering>true</filtering>
             </resource>
         </resources>
     </build>
 
-    <profiles>
-        <profile>
-            <id>testsSkipJdk6</id>
```

Comment:
Does it make sense to support such historical artifact?

Reply:
probably that is why this profile is being removed...

Reply:
Yes. My fault, I somehow did not realize that this is "red". :)